chain-signatures/node/src/metrics.rs

```diff
@@ -335,6 +335,51 @@ pub(crate) static FAILED_SEND_ENCRYPTED_LATENCY: Lazy<HistogramVec> = Lazy::new(
     .unwrap()
 });
 
+pub(crate) static NUM_TOTAL_HISTORICAL_SIGNATURE_GENERATORS: Lazy<IntGaugeVec> = Lazy::new(|| {
```

Comment:
Should we rename this to `SIGNATURE_GENERATOR`, `NEW_SIGNATURE_GENERATOR`, or something similar?
We should not expose how it is represented on Grafana in code. It can be reused, it can be per hour, not historical. It can be combined with other data and not be `TOTAL`.

I would rename all new and existing metrics so they represent what they are, and not how they are shown on in Grafana.